src/main/scala/com/fulcrumgenomics/umi/VanillaUmiConsensusCaller.scala

```diff
@@ -60,19 +68,69 @@ case class VanillaUmiConsensusCallerOptions
   minInputBaseQuality: PhredScore         = DefaultMinInputBaseQuality,
   minConsensusBaseQuality: PhredScore     = DefaultMinConsensusBaseQuality,
   minReads: Int                           = DefaultMinReads,
-  requireConsensusForBothPairs: Boolean   = DefaultRequireConsensusForBothPairs
+  requireConsensusForBothPairs: Boolean   = DefaultRequireConsensusForBothPairs,
+  producePerBaseTags: Boolean             = DefaultProducePerBaseTags
 )
 
-/** Stores information about a read to be fed into a consensus. */
+/**
+  * Object that encapsulates the various consensus related tags that are added to consensus reads
+  * at both the per-read and per-base level.
+  *
+  * Currently only contains tags for single-strand consensus reads, but with a view to using the following
+  * names for consistency if/when we add duplex calling:
+  *     Value                 AB  BA  Final
+  *     ===================== ==  ==  =====
+  *     bases                 ac  bc  bases
+  *     quals                 aq  bq  quals
+  *     per-read-depth        aD  bD  cD
+  *     per-read-min-depth    aM  bM  cM
+  *     per-read-error-count  aE  bE  cE
+  *     per-base-depth        ad  bd  cd
+  *     per-base-error-count  ae  be  ce
```

Comment:
I was waiting to see if you agreed with it!  Yes, let's add an update to the wiki page to the todo list.